app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render partial: "partials/login_sign_up", locals: {title: "TRANSACTIONS", no_auth: true}%>
+
+<div class="total_expenses flex general_text">
+    <span>Total amount spent in this category:</span> <span>$<%= @total_amount %></span>
+</div>
+
+<div class="expenses_container flex_column">
+    <% @expenses.each do |exp|%>
+        <div class="expense_container flex center_horizontal general_text">
+            <span class="expense_name"><%= exp.expense.name%></span>
+            <span class="expense_amount">$<%= exp.expense.amount%></span>
+        </div>
```

Comment:
- [ ]  Here we need to see when the transaction happened. Please the date when it is created.

### Your Design
![Screenshot from 2023-02-25 04-41-47](https://user-images.githubusercontent.com/26767383/221318922-4e34c718-e3c4-4def-9ef6-bf6b405cb5d3.png)


### Given design

![Screenshot from 2023-02-25 04-42-14](https://user-images.githubusercontent.com/26767383/221318942-7184a138-6a9c-4085-9410-56d70dbb766e.png)

Reply:
Oh okay, I will add that immediately. Thank you